.travis/setup_cassandra.sh

```diff
@@ -5,5 +5,5 @@ source ./versions.sh
 CASSANDRA_BASE=apache-cassandra-$CASSANDRA_VERSION
 
 sudo rm -rf /var/lib/cassandra/*
-curl http://www.us.apache.org/dist/cassandra/$CASSANDRA_VERSION/$CASSANDRA_BASE-bin.tar.gz | tar xz
+curl http://apache.spinellicreations.com/cassandra/$CASSANDRA_VERSION/$CASSANDRA_BASE-bin.tar.gz | tar xz
```

Comment:
What actually happened is that some day, all of a sudden, version 2.1.{3 I think} just disappeared from the main repo, without any notice... I used this mirror which still had it but after a while it disappeared too. Turns out we don't have a choice but to update to 2.1.{4,5}, which we did already in master.